### PR TITLE
Add Moto G (falcon)

### PR DESCRIPTION
**Pull request by yagyaxt1068**
Add the first-generation Moto G (falcon) to device manifests.

### CODE REVIEW

File: manifests/motorola_falcon.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <project path="device/motorola/falcon" name="android_device_motorola_falcon" remote="los" revision="cm-14.1" />
```
Comment (jbruechert) [inline manifests/motorola_falcon.xml:3]: The "los" remote already sets the revision to a default of "cm-14.1". Please don't unnecessarily override it.

Comment (jbruechert) [reply manifests/motorola_falcon.xml:3]: It's also likely that you have to make changes in the device repository, so please fork and point the manifest to your own repository.

File: manifests/motorola_falcon.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  <project path="device/motorola/msm8226-common" name="android_device_motorola_msm8226-common" remote="los" revision="cm-14.1" />
+  <project path="vendor/motorola" name="proprietary_vendor_motorola" remote="them" revision="cm-14.1" />
```
Comment (jbruechert) [inline manifests/motorola_falcon.xml:5]: The manifest is missing a kernel repository, please add one containing your kernel config changes.

**Comment by Vince1171** [comment]
closed as it only uses AOSP/LineageOS/vendor repo without any changes
if the port works without any changes, plz ping me so I reopen this PR

**Pull request closed without merging**